apps/condo/domains/acquiring/schema/Payment.test.js

```diff
@@ -274,7 +290,7 @@ describe('Payment', () => {
                 })
             })
         })
-        test('cannot link to multipayment with different integration', async () => {
+        test.skip('cannot link to multipayment with different integration', async () => {
```

Comment:
?

Reply:
This test was created when link "Payment -> MutliPyament" was required, but then we decided to make it not required to allow external payments in our system...

I'll fix all validations tests of Payment / MultiPayment in another task (https://doma.atlassian.net/browse/DOMA-1452)